conf/patches/gke-helmfile-deployment.sh

```diff
@@ -1,9 +1,12 @@
+#!/bin/bash
+
 for filename in /conf/helmfile.d/*.yaml; do
-    echo ${filename}
-    #echo 'grep "\- name: " ${filename} | grep -m1 -v "\- name: \"stable\"" | awk {print \$3} | sed s/^\"\(.\+\)\"$/\1/'
     deployment_name=$(grep "\- name: " ${filename} | grep -m1 -v "\- name: \"stable\"" | awk '{print $3}' | sed 's/^\"\(.\+\)\"$/\1/')
-    echo ${deployment_name}
-    until helmfile --selector name=${deployment_name} sync; do
+    retries=3
+    for ((i=0; i<retries; i++)); do
+        helmfile --selector name=${deployment_name} sync
```

Comment:
Instead, consider the retry command that ships with geodesic.

Reply:
https://github.com/kamilsk/retry

Reply:
Oh, that looks nice! I'll check this out.